### ISSUE TITLE
From china user

### ISSUE THREAD

**Issue by sunjay-lee** [lgsvl/simulator#1346]
Since latest update , the SVL has to be run under login_in state. But in China I find it hard to connect to the server or maybe some other reasons lead to this. It is not easy to connect , since the button always comes from "connecting" to "offline" but not "started". See the pictures above, will SVL allows more offline functions in the future? Or would you provide a little more assistence for Chinese users so that connecting is no longer a problem for us?
![lALPDiCpuli555HNAe3NAhA_528_493](https://user-images.githubusercontent.com/59258176/113830928-a94be780-97b9-11eb-882d-b6d24d3e6d79.png)
![lALPDhmOvQ8tFlnNAeXNAhA_528_485](https://user-images.githubusercontent.com/59258176/113830939-acdf6e80-97b9-11eb-941b-d3a94968b107.png)

**Comment by EricBoiseLGSVL** [comment]
@sunjay-lee Yes and we apologize.  We haven't been able to solve this yet but we are looking at solutions.
